standards.md

```diff
@@ -0,0 +1,83 @@
+# Standards for development within PSD
```

Comment:
Does this document define the standards, or is it a template we should copy to our other repos?

Reply:
I was thinking it can define standards for us. We could move it to confluence? I prefer markdown to the confluence format but I think it matters most that we feel we can update/contribute easily and quickly; so wherever that is.